Documentation/WritingReST/Hyperlinks.rst

```diff
@@ -58,6 +58,13 @@ by adding a label before the section header::
 
 How to create links is described in more detail in the next sections.
 
+:ref:`Preventing links <preventing-links>`
```

Comment:
IMO this shouldn't be the first thing to mention in a section about linking. ;-)

Since we already have a more thorough description at the end of this section we can skip it here.

Reply:
To me the first section looked like a Table of Contents. If you use it like links to most important parts of the further documentation, then we should skip it, yes.

Reply:
Ah indeed, just had another look at the rendered result. Still we should move this after the 3 existing entries here, again because it's less important than actual linking.

Reply:
yes, it should be already at the last position ;)

Reply:
I already had my coffee, I swear. Thanks for pointing out the obvious to me. :+1: